_extras/snippets_library/UCL_sge/14/module_avail.snip

```diff
@@ -1,49 +1,19 @@
 
----------------------------------------------- /usr/share/Modules/modulefiles ----------------------------------------------
-dot         module-git  module-info modules     mpt/2.16    null        perfboost   perfcatcher use.own
+---------------------- /shared/ucl/apps/modulefiles/core -----------------------
+gerun             ops-tools/1.1.0   screen/4.2.1      userscripts/1.3.0
+lm-utils/1.0      ops-tools/2.0.0   userscripts/1.0.0 userscripts/1.4.0
+mrxvt/0.5.4       rcps-core/1.0.0   userscripts/1.1.0
+ops-tools/1.0.0   rlwrap/0.43       userscripts/1.2.0
+
+------------------ /shared/ucl/apps/modulefiles/applications -------------------
+abaqus/2017
+adf/2014.10
+afni/20151030
+afni/20181011
+amber/14/mpi/intel-2015-update2
+amber/14/openmp/intel-2015-update2
+amber/14/serial/intel-2015-update2
+amber/16/mpi/gnu-4.9.2
+
+[truncated]
```

Comment:
Use 
```suggestion
[output truncated]
```
to keep consistency between all the episodes.

Reply:
Good point, will do.

Reply:
Fixed in commit 4ee0961